maestro-ios-xctest-runner/maestro-driver-iosUITests/Routes/Handlers/InputTextRouteHandler.swift

```diff
@@ -61,4 +63,18 @@ struct InputTextRouteHandler : HTTPHandler {
         let errorData = Data(jsonString.utf8)
         return HTTPResponse(statusCode: HTTPStatusCode.badRequest, body: errorData)
     }
+    
+    private func waitUntilKeyboardIsPresented(appId: String?) async {
+        let timeout = Date().addingTimeInterval(2) // Set the timeout for 2 second from now
```

Comment:
I'm a bit worried that the 2 seconds delay will have an impact on various flow runs. Do you know how often the the keyboard is not found and maestro waits for 2 seconds?

Reply:
In all of my test cases it was available after the first iteration (meaning only 200ms delay), I think it is safe to decrease the timeout to 1s